packages/eas-cli/src/commands/submit.ts

```diff
@@ -87,13 +99,11 @@ export default class BuildSubmit extends EasCommand {
 
     track: flags.enum({
       description: 'The track of the application to use',
-      default: 'internal',
```

Comment:
Removing the `default` property affects command `--help` output. Maybe we can update the description, or somehow take the value from some single source of truth, and also provide them to the `options` variable down below. 🤷

Reply:
another possible way to handle this is to make `--profile` and other submission options (save for url, etc, which are not in profile) mutually exclusive. so if you profile `--profile` and `--track` then we error. this might be a little bit annoying for users though...